examples/connectivity/plot_adhd_covariance.py

```diff
@@ -20,13 +20,11 @@
 def plot_matrices(cov, prec, title):
     """Plot covariance and precision matrices, for a given processing. """
 
+    # Compute sparsity pattern
+    sparsity = prec == 0
```

Comment:
I know that it was the code but I think that `sparsity = (prec == 0)` is clearer.